packages/regl-worldview/src/commands/GLText.js

```diff
@@ -53,11 +54,9 @@ type FontAtlas = {|
 
 // Font size used in rendering the atlas. This is independent of the `scale` of the rendered text.
 const FONT_SIZE = 40;
-const BUFFER = 10;
 const MAX_ATLAS_WIDTH = 512;
 const SDF_RADIUS = 8;
-const CUTOFF = 0.25;
-const OUTLINE_CUTOFF = 0.6;
+const CUTOFF = 0.75;
```

Comment:
Rather than changing the cutoff, I think it might be better to keep the buffer around characters in the atlas, to avoid them accidentally bleeding into each other. You would keep your changes to destOffsets/srcWidths, but revert the changes to rowHeight and dx when building the atlas, and then just offset info.x/info.y by the buffer amount.

Reply:
nice suggestion -- done!